git_greppers/utils.sh

```
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# ANSI color codes
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
```

Shellcheck (0.10.0)

[warning] BLUE appears unused. Verify use (or export if used externally).

(SC2034)